tests/test_ops/test_tensorrt.py

```diff
@@ -402,3 +402,88 @@ def test_deform_conv():
     if os.path.exists(trt_file):
         os.remove(trt_file)
     assert torch.allclose(pytorch_results, trt_results)
+
+
```

Comment:
Add the following decorator instead of creating a `params_list` would be clearer. 
```python
@pytest.mark.parametrize('mode', ['bilinear', 'nearest'])
@pytest.mark.parametrize('padding_mode', ['zeros', 'border', 'reflection'])
@pytest.mark.parametrize('align_corners', [True, False])
def test_grid_sample(mode, padding_mode, align_corners):
```